src/pokemon_sprite_visualizer.c

```diff
@@ -398,16 +398,16 @@ const u8 gBattleBackgroundTerrainNames[][26] =
     [BATTLE_TERRAIN_PLAIN]      = _("NORMAL - PLAIN           "),
 };
 //Function declarations
-static void PrintDigitChars(struct PokemonDebugMenu *data);
-static void SetUpModifyArrows(struct PokemonDebugMenu *data);
-static void UpdateBattlerValue(struct PokemonDebugMenu *data);
+static void PrintDigitChars(struct PokemonSpriteVisualizer *data);
+static void SetUpModifyArrows(struct PokemonSpriteVisualizer *data);
+static void UpdateBattlerValue(struct PokemonSpriteVisualizer *data);
 static void ValueToCharDigits(u8 *charDigits, u32 newValue, u8 maxDigits);
-static bool32 TryMoveDigit(struct PokemonDebugModifyArrows *modArrows, bool32 moveUp);
-static void CB2_Debug_Runner(void);
-static void ResetBGs_Debug_Menu(u16);
-static void Handle_Input_Debug_Pokemon(u8);
-static void ReloadPokemonSprites(struct PokemonDebugMenu *data);
-static void Exit_Debug_Pokemon(u8);
+static bool32 TryMoveDigit(struct PokemonSpriteVisualizerModifyArrows *modArrows, bool32 moveUp);
+static void CB2_Pokemon_Sprite_Visualizer_Runner(void);
+static void ResetBGs_Pokemon_Sprite_Visualizer(u16);
+static void Handle_Input_Pokemon_Sprite_Visualizer(u8);
+static void ReloadPokemonSprites(struct PokemonSpriteVisualizer *data);
+static void Exit_Pokemon_Sprite_Visualizer(u8);
```

Comment:
I don't like the underscore usage for these functions originally, so I would recommend something like this:
```suggestion
static void CB2_PokemonSpriteVisualizerRunner(void);
static void ResetBGs_PokemonSpriteVisualizer(u16);
static void HandleInput_PokemonSpriteVisualizer(u8);
static void ReloadPokemonSprites(struct PokemonSpriteVisualizer *data);
static void Exit_PokemonSpriteVisualizer(u8);
```